python/setup.py

```diff
@@ -258,18 +255,12 @@ def get_packages(self):
 # the change in the matching section of requirements/requirements.txt
 if setup_spec.type == SetupType.RAY:
     setup_spec.install_requires = [
-        "attrs",
-        "click >= 7.0",
-        "dataclasses; python_version < '3.7'",
-        "filelock",
-        "grpcio >= 1.28.1",
-        "jsonschema",
-        "msgpack >= 1.0.0, < 2.0.0",
-        "numpy >= 1.16; python_version < '3.9'",
-        "numpy >= 1.19.3; python_version >= '3.9'",
-        "protobuf >= 3.15.3",
-        "pyyaml",
-        "redis >= 3.5.0",
+        "attrs", "click >= 7.0", "dataclasses; python_version < '3.7'",
+        "filelock", "grpcio >= 1.28.1", "jsonschema",
+        "msgpack >= 1.0.0, < 2.0.0", "numpy >= 1.16; python_version < '3.9'",
+        "numpy >= 1.19.3; python_version >= '3.9'", "protobuf >= 3.15.3",
+        "pyyaml", "redis >= 3.5.0", "aioredis < 2", "aiosignal", "frozenlist",
+        ""
```

Comment:
why is there an empty thing here?

Reply:
Maybe a mistake. Let me remove it

Reply:
Fixed

Reply:
Not the biggest fan of the dependency shift but I guess we can move forward with it.